code/game/objects/items/stacks/medical_vr.dm

```diff
@@ -0,0 +1,21 @@
+/obj/item/stack/medical/advanced
+	icon = 'icons/obj/stacks_vr.dmi'
+
+/obj/item/stack/medical/advanced/New()
+	..()
+	update_icon()
+	return
+
+/obj/item/stack/medical/advanced/update_icon()
+	if(amount <= 2)
+		icon_state = initial(icon_state)
```

Comment:
Make all this into a switch statement. Use `[lower bound] to [upper bound]`. It's just faster.